src/net/atlas/db.rs

```diff
@@ -112,6 +114,21 @@ impl AtlasDB {
         )
         .map_err(db_error::SqliteError)?;
 
+        if let Some(attachments) = genesis_attachments {
+            let now = util::get_epoch_time_secs() as i64;
```

Comment:
Where is `created_at` used? If I decide to boot a Stacks 2.0 node a couple months after mainnet -- won't all my names be 2 months newer than the first node? Maybe this value should be based on the burnchain block time?

Reply:
It doesn't matter, the `created_at` is just used for pruning uninstantiated attachments.